pre_commit/commands/run.py

```diff
@@ -183,4 +182,11 @@ def run(runner, args, write=sys_stdout_write_wrapper, environ=os.environ):
             if not repo_hooks:
                 write('No hook with id `{0}`\n'.format(args.hook))
                 return 1
+       
+        if args.hook_stage:
+            repo_hooks = [
+                (repo, hook) for repo, hook in repo_hooks
+                if not hook['stages'] or args.hook_stage in hook['stages']
+            ]
```

Comment:
Right. And that sample output request would be in a test. The only problem now is that I can't get the previous tests to work. Once they are working, then I will be happy to test this new functionality.